packages/runtime/src/runtimes/react/index.tsx

```diff
@@ -44,66 +43,106 @@ import {
   parseBreakpointsInput,
 } from '../../state/modules/breakpoints'
 
-export const storeContextDefaultValue = ReactPage.configureStore()
+export class ReactRuntime {
+  // TODO: the static methods here are deprecated and only keep here for backward-compatibility purpose.
+  // We will remove them when we release a new breaking change.
+  // ------------------ Deprecated API ------------------ //
```

Comment:
We might even want to add a JSDoc deprecated tag so that it shows up in users' VSCode once we actually do release a stable version of breakpoints and deprecate this usage, like so:

```
/**
 * @deprecated The ReactRuntime singleton was deprecated in v0.9.0, instantiate your own runtime instead. See here for more details...
 */
```

Reply:
I thought about that, but I intentionally did not put the deprecated tag because we don't the documentation ready for the new API pattern yet.

We can put the deprecated tag once we have all the documentation updated and also once we have a guide on how to migrate to the new API pattern.